package.json

```diff
@@ -61,6 +61,7 @@
     "codecov": "2.3.0",
     "concurrently": "3.5.0",
     "cross-env": "3.2.4",
+    "cypress": "1.0.2",
```

Comment:
Should we add an npm script to run these e2e tests?

Reply:
Yeah, `npx cypress run` was simple enough, but since it's not yet available for everyone, a script could be helpful.

Reply:
Also just consistency / consolidation. We have `npx jest` and `npx eslint` as well, with npm scripts which are effectively just that.